### ISSUE TITLE
The news posts from the QAT Blog need to be ported to the osu!wiki

### ISSUE THREAD

**Issue by TPGPL** [ppy/osu-wiki#1014]
http://osuqat.tumblr.com

**Comment by Joehuu** [comment]
@MegaApplePi Was there a special technique when you were porting the old news posts from https://osunews.tumblr.com/ or the formatting was done manually with raw text (copy paste)?

And how about maintaining the comments/date of that post?

**Comment by MegaApplePi** [comment]
It was some kind of process. I opted copy the HTML from tumblr then have a [website](https://domchristie.github.io/turndown/) process them individually to get me the Markdown. After that, I had to manually save any images that did not 404, fix pretty much all of the URLs (tumblr has external URLs go through themselves first), clean up any excessive white space (though I may have not done this for all of them), fix any malformed HTML that was left behind, and fix any mistakes the website made while processing them. I also needed to add the YAML data to each news post as well.

Afaik, comments were handled by disqus, which somehow worked by using the YAML data in the news posts... I guess?? Comments are now being handled by osu-web's comment system, so I don't know how it would work (that is, if it would still try to transfer them over, if it ever did that). Dating of the posts was based on the date listed in tumblr (found in the archive), while time was assumed to be 00:00 at UTC-0 (both of which, may have been a not-so-great idea).

---

Though, since the QAT tumblr doesn't seem to have as many posts that the news did, I could work on porting them over. I just need to know where they go... into `news/`?
